crates/orchestrator/src/lib.rs

```diff
@@ -267,7 +267,7 @@ where
         self.pub_posted.insert(node_index);
 
         // The guess is the first extra 8 bytes are from orchestrator serialization
-        pubkey.drain(..8);
+        pubkey.drain(..12);
```

Comment:
I also changed this in my PR haha, would be better to also change the comment from 8 to 12.

Reply:
Haha yeah, I will look into why we need this padding soon. I'll change this comment in the meantime.